### PR TITLE
#07 Feat: Add Tag Component 

### PR DESCRIPTION
**Pull request by clever-cr**
### Task Description _(❗️Required)_

Created tag component to be used in other sections
### Type of Task

- [ ] fix
- [ ] chore
- [x] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [x] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_
https://www.notion.so/apeunit/Tag-component-50ec6069c87c4157b816921768a5fa19

### CODE REVIEW

File: components/ui/Tag.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const Tag = ({ content }) => {
+  return (
+    <div>
+      <p className="font-neuf text-lg text-primaryGray-400 font-normal -tracking-wide bg-primaryGray-200 rounded-base px-2.5 py-sm  w-fit">
```
Comment (mercihabam) [inline components/ui/Tag.jsx]: `font-normal` className is not actually needed since it doesn't change anything on the text, it's good to remove it.

**Review by mercihabam — CHANGES_REQUESTED** [review]
## Congrats for what you've done

Just on thing to have a look on, kindly refer to the comment I left.

File: components/ui/Tag.jsx (hunk 1 above)
Comment (Odile078) [inline components/ui/Tag.jsx:4]: You forgot to add a custom line-height and the font-size for the small screens.

File: components/ui/Tag.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const Tag = ({ content }) => {
+  return (
+    <div>
+      <p className="font-neuf text-lg text-primaryGray-400  -tracking-wide  bg-primaryGray-200  rounded-base  px-2.5  py-sm w-fit">
+        {content}
+      </p>
+    </div>
```
Comment (alain-kubwayo) [inline components/ui/Tag.jsx]: You introduced an unnecessary tag. `<div>` doesn't have any semantic meaning to your code and your code should still work without the outer div.

File: components/ui/Tag.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const Tag = ({ content }) => {
+  return (
+    <div>
+      <p className="font-neuf text-lg text-primaryGray-400  -tracking-wide  bg-primaryGray-200  rounded-base  px-2.5  py-sm w-fit">
```
Comment (alain-kubwayo) [inline components/ui/Tag.jsx:4]: The padding should be adjusted according to the design like so, `padding: 3px 10px;`

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job!
```
Minor adjustments and you're good to go.

**Review by alain-kubwayo — APPROVED** [review]
Looks great! 🚀

**Review by Odile078 — APPROVED** [review]
LGTM

**Review by Lily-Lilian — APPROVED** [review]

**Review by mercihabam — APPROVED** [review]

**Pull request merged**